kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -355,9 +355,15 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
 
   private def getEngineSpace(engine: Engine): String = {
     val serverSpace = fe.getConf.get(HA_NAMESPACE)
+    val owner = if (engine.getSharelevel == "GROUP") {
```

Comment:
Reference `EngineRef`, using the scala syntax
```
engine.getSharelevel match {
    case XXX =>
       // do something
    case  _ =>
       // do something
}
```

Reply:
the SERVER mode is not correct either

Reply:
> the SERVER mode is not correct either

Could this be in another PR?